symengine/ruby/lib/symengine/integer.rb

```diff
@@ -0,0 +1,93 @@
+class Fixnum
```

Comment:
You shouldn't modify built-in classes of a programming language even if it is allowed.
Take a look at class coercion in Ruby.